### PR TITLE
Ems 114005589 - Add lesson feedback content provider.

### PR DESCRIPTION
**Pull request by eschlange**
- Add new content provider for lesson feedback.
- Add migration and update schema.
- Add new model and spec.
- Add content provider view.
- Updated participant association to new model.
- Added css updates for content provider.
- Added seed data for local testing.

[#114005589] [#114005545]

### CODE REVIEW

File: app/assets/javascripts/content_providers/lesson_feedback.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+(function(context) {
+  'use strict';
+
+  var LessonFeedback = {
+    clearRating: function() {
+      $("#star1").removeClass("star-selected");
```
Comment (michaelwehrley) [inline app/assets/javascripts/content_providers/lesson_feedback.js]: You could write:
```javascript
$('#star1, #star2, #star3, #star4, #star5')
  .removeClass('star-selected');
```

File: app/views/content_providers/lesson_feedback/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<h2>Lesson Feedback</h2>
+
+<%= form_for current_participant.lesson_feedbacks.build, url: create_path, method: :post, html: { role: :form } do |f| %>
+  <%= hidden_field :lesson_feedback, :bit_core_content_module_id, value: bit_core_content_module_id %>
+
+  <div class="form-group">
+    <%= f.label :rating, "Please rate today's lesson." %>
+    <%= hidden_field :lesson_feedback, :rating, value: 0 %>
```
Comment (michaelwehrley) [inline app/views/content_providers/lesson_feedback/new.html.erb]: Do we want the value `nil`? When we calc the ratings later it might be skewed.

Comment (eschlange) [inline app/views/content_providers/lesson_feedback/new.html.erb]: It is allowed to leave the rating blank, but to include comments. My idea was to allow for nil, we can always take that into consideration when calculating ratings.

**Comment by michaelwehrley** [comment]
I just have that question about the rating.  Otherwise, things look great.  Thanks!

**Comment by eschlange** [comment]
@michaelwehrley Thanks! I'll resolve the conflicts before merging.

File: app/views/content_providers/lesson_feedback/new.html.erb (hunk 2 above)
Comment (michaelwehrley) [inline app/views/content_providers/lesson_feedback/new.html.erb]: @eschlange To seems that if we need to remember when we calculate to remove all 0s that could be problematic.  To me the default rating that exists i.e., `nil` would/does imply that it wasn't rated and I would keep that consistency.

Comment (eschlange) [inline app/views/content_providers/lesson_feedback/new.html.erb]: I agree, I'll remove the default value. When I do that, the database value
will be NULL.

Regards,
Eric

**Pull request merged**